runtime/src/bank/partitioned_epoch_rewards/calculation.rs

```diff
@@ -457,6 +523,7 @@ impl Bank {
             stake_history,
             stake_delegations,
             cached_vote_accounts,
+            ..
```

Comment:
We can delete this dotdot ignore since we fully unpack the struct. right?

Reply:
Done